core/src/main/java/org/apache/iceberg/util/ThreadPools.java

```diff
@@ -82,6 +87,20 @@ public static ExecutorService getDeleteWorkerPool() {
     return DELETE_WORKER_POOL;
   }
 
+  /**
+   * A shared {@link ScheduledExecutorService} that REST catalogs can use for refreshing their
+   * authentication data.
+   */
+  public static ScheduledExecutorService getAuthRefreshPool() {
```

Comment:
```suggestion
  public static ScheduledExecutorService authRefreshPool() {
```

I'm aware that we already have some methods in this class with a `get` prefix, but those methods have been added at the very beginning and haven't been renamed